src/main/java/fi/otavanopisto/kuntaapi/server/index/ServiceSearcher.java

```diff
@@ -38,9 +50,12 @@
   @Inject
   private IndexReader indexReader;
 
-  public SearchResult<ServiceId> searchServices(OrganizationId organizationId, String text, ServiceSortBy sortOrder, SortDir sortDir, Long firstResult, Long maxResults) {
-    BoolQueryBuilder query = boolQuery()
-      .must(queryStringQuery(text));
+  public SearchResult<ServiceId> searchServices(OrganizationId organizationId, ElectronicServiceChannelId electronicServiceChannelId, PhoneServiceChannelId phoneServiceChannelId, PrintableFormServiceChannelId printableFormServiceChannelId, ServiceLocationServiceChannelId serviceLocationServiceChannelId, WebPageServiceChannelId webPageServiceChannelId, String text, ServiceSortBy sortOrder, SortDir sortDir, Long firstResult, Long maxResults) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 11 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)